closed/autoconf/custom-hook.m4

```diff
@@ -455,9 +455,16 @@ AC_DEFUN([OPENJ9_PLATFORM_SETUP],
     OPENJ9_BUILD_MODE_ARCH=arm_linaro
     OPENJ9_LIBS_SUBDIR=default
   elif test "x$OPENJ9_CPU" = xaarch64 ; then
-    OPENJ9_PLATFORM_CODE=xr64
-    if test "x$COMPILE_TYPE" = xcross ; then
-      OPENJ9_BUILD_MODE_ARCH="${OPENJ9_BUILD_MODE_ARCH}_cross"
+    if test "x$OPENJ9_BUILD_OS" = xlinux ; then
+      OPENJ9_PLATFORM_CODE=xr64
+      if test "x$COMPILE_TYPE" = xcross ; then
+        OPENJ9_BUILD_MODE_ARCH="${OPENJ9_BUILD_MODE_ARCH}_cross"
+      fi
+    elif test "x$OPENJ9_BUILD_OS" = xmacosx ; then
+      OPENJ9_PLATFORM_CODE=or64
+      OPENJ9_BUILD_OS=osx
+    else
```

Comment:
I'd prefer to see `OPENJ9_BUILD_OS` set above (see line 403) in one place. The tests here should test `OPENJDK_BUILD_OS` instead of `OPENJ9_BUILD_OS`.

Reply:
The code is taken from the code for x86-64 above.
Should I rewrite the x86-64 part, too?

Reply:
Changed the tests from `OPENJ9_BUILD_OS` to `OPENJDK_BUILD_OS`.

Reply:
What you have now should work; perhaps the refactoring I suggest above can be deferred.